src/js/visual/csr/utils.js

```diff
@@ -276,16 +297,12 @@ ripe.CsrUtils.loadFBX = async function(path) {
  * Loads a GLTF/GLB file.
  *
  * @param {String} path Path to the file. Can be local path or an URL.
- * @param {Boolean} useDracoLoader Dictates if it should use draco loader to decode the file.
+ * @param {THREE.DRACOLoader} dracoLoader A Draco loader instance.
  * @returns {THREE.Object3D} The loaded GLTF/GLB file.
  */
-ripe.CsrUtils.loadGLTF = async function(path, useDracoLoader = true) {
+ripe.CsrUtils.loadGLTF = async function(path, dracoLoader = null) {

```

Comment:
```suggestion
ripe.CsrUtils.loadGLTF = async function(path, dracoLoader = null) {
```

Just to confirm this ` = null` is a convention to enforce that `dracoLoader` can come as null, right?

Reply:
Yes